sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/ShareDirectoryAsyncClient.java

```diff
@@ -659,7 +659,7 @@ public ShareDirectoryAsyncClient getSubdirectoryClient(String subdirectoryName)
             return withContext(context -> forceCloseAllHandlesWithTimeout(recursive, null, context)
                 .reduce(new CloseHandlesInfo(0, 0),
                     (accu, next) -> new CloseHandlesInfo(accu.getClosedHandles() + next.getClosedHandles(),
-                        accu.getFailedHandles() + next.getFailedHandles())));
+                        accu.getFailedToCloseHandles() + next.getFailedToCloseHandles())));
```

Comment:
Does this naming align with the other languages? I believe I saw different in Python.

Reply:
This name aligns as much as possible without making the other property getClosedHandles sound unaligned